plugins/catalog-backend/src/database/Database.ts

```diff
@@ -28,7 +28,7 @@ export class Database {
   constructor(private readonly database: Knex) {}
 
   async addOrUpdateComponent(component: AddDatabaseComponent): Promise<void> {
-    await this.database.transaction(async (tx) => {
+    await this.database.transaction(async tx => {
```

Comment:
oh no, has our prettier started flaking back and forth about this parentheses thing :)